### DISCUSSION TITLE
How to exclude search results?

### DISCUSSION THREAD

**Discussion by RussKie** [community/community#23330] [New to GitHub]
<p>About 40% of commits in <a href="https://github.com/dotnet/winforms/" rel="nofollow noopener">https://github.com/dotnet/winforms/</a> repo is done by a bot (<a href="https://github.com/dotnet/winforms/pulls?q=is%3Apr+sort%3Aupdated-desc+is%3Aclosed+author%3Aapp%2Fdotnet-maestro" rel="nofollow noopener">example</a>).</p>
<p>I want to see all PRs from humans and exlcude the automated PRs. How can I do that?</p>

**Comment by SethClydesdale** [comment, marked as answer]
<p>Hello <a class="mention" href="/russkie">@russkie</a>,</p>
<p>You can exclude certain subsets by prefixing the qualifier with a “-”. So <a href="https://github.com/dotnet/winforms/pulls?utf8=%E2%9C%93&amp;q=is%3Apr+sort%3Aupdated-desc+is%3Aclosed+-author%3Aapp%2Fdotnet-maestro+" rel="nofollow noopener">-author:app/dotnet-maestro</a> will exclude all results where the author is the bot. You can read more about the search syntax <a href="https://help.github.com/en/articles/understanding-the-search-syntax#exclude-certain-results" rel="nofollow noopener">here</a>.</p>

**Comment by RussKie** [comment]
<p>Awesome! Thank you.</p>
<p>I wish the same page was linked from <a href="https://help.github.com/en/articles/searching-issues-and-pull-requests" rel="nofollow noopener">https://help.github.com/en/articles/searching-issues-and-pull-requests</a> or from the actual “advanced search” link</p>
<p><img src="https://user-images.githubusercontent.com/4403806/181076168-983944e4-1873-4e37-922b-8290e7cad528.png" alt="github-search.png" data-base62-sha1="j1kx0cgU1Dai0AAz8U2Q0GZPpil" width="394" height="322"></p>

**Comment by Neustradamus** [comment]
<p>It does not work, examples:</p>
<ul>
<li>
<a href="https://github.com/search?q=org%3Agithub+test+-author%3Agithub%2Fgit-msysgit&amp;type=Code" rel="nofollow noopener">https://github.com/search?q=org%3Agithub+test±author%3Agithub%2Fgit-msysgit&amp;type=Code</a><br>
or<br>
- <a href="https://github.com/search?q=org%3Agithub+test+-org%3Agithub%2Fgit-msysgit&amp;type=Code" rel="nofollow noopener">https://github.com/search?q=org%3Agithub+test±org%3Agithub%2Fgit-msysgit&amp;type=Code</a>
</li>
</ul>
<p>github/git-msysgit always in search results.</p>
